pom.xml

```diff
@@ -349,42 +350,45 @@
         </executions>
       </plugin>
       <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>3.3.0</version>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version>
         <executions>
           <execution>
-            <id>create-runnable-jar</id>
+            <id>create-shaded-jar</id>
             <phase>package</phase>
             <goals>
-              <goal>single</goal>
+              <goal>shade</goal>
             </goals>
             <configuration>
-              <descriptors>
-                <descriptor>src/assembly/driver.xml</descriptor>
-              </descriptors>
-              <archive>
-                <manifest>
-                  <mainClass>org.fcrepo.migration.PicocliMigrator</mainClass>
-                </manifest>
-              </archive>
-            </configuration>
-          </execution>
-          <execution>
-            <id>create-distribution</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-            <configuration>
-              <finalName>migration-utils</finalName>
-              <descriptors>
-                <descriptor>src/assembly/zip.xml</descriptor>
-              </descriptors>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <finalName>${project.artifactId}-${project.version}-driver</finalName>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <manifestEntries>
+                    <Main-Class>${project.main.class}</Main-Class>
+                  </manifestEntries>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                  <resource>conf/fedora3.xml</resource>
```

Comment:
Building from main I don't see this in the artifact, is this something we had missed from before?

Reply:
That is a good question, I was looking at the assembly plugin `zip.xml` file which appears to be [including](https://github.com/fcrepo-exts/migration-utils/blob/main/src/assembly/zip.xml#L7-L10) the contents of the `conf` directory in the packaged result. But if this file is not necessary I can remove that.